### DISCUSSION TITLE
Edges between services not forming between proxies

### DISCUSSION THREAD

**Discussion by jnapprogs** [linkerd/linkerd2#4946] [General]
I just started using Linkerd after following the **Getting Started** guide.  In the guide, the services had edges between each other when I ran the command `linkerd edges deploy -n emojivoto`.  When I injected Linkerd into my services, running `kubectl get po` shows that the sidecars are running.  The Linkerd dashboard also verifies that my services are meshed; however, when I run `linkerd edges deploy -n`, I can see that there is an edge from **linkerd-prometheus** to my two services but there is none between each other.  I was wondering if I'm missing some sort of configuration.

I'm currently using Amazon EKS to host my cluster.

**Comment by ihcsim** [comment]
Assuming that there is constant traffic between your services, is the traffic encrypted without Linkerd? If yes, Linkerd can't MITM your traffic and simply treats it as TCP. Usually, no additional configuration is required.
